### PR TITLE
error logs to stderr and bump oryx

### PR DESCRIPTION
**Pull request by patricklee2**

### CODE REVIEW

File: template-apache/Dockerfile [hunk 1]
```diff
@@ -34,6 +34,16 @@ ENV PATH ${PATH}:/home/site/wwwroot
 RUN sed -i 's!ErrorLog ${APACHE_LOG_DIR}/error.log!ErrorLog /dev/stderr!g' /etc/apache2/apache2.conf 
 RUN sed -i 's!User ${APACHE_RUN_USER}!User www-data!g' /etc/apache2/apache2.conf 
 RUN sed -i 's!User ${APACHE_RUN_GROUP}!Group www-data!g' /etc/apache2/apache2.conf 
+RUN sed -i 's!CustomLog ${APACHE_LOG_DIR}/other_vhosts_access.log vhost_combined!CustomLog /dev/stderr vhost_combined!g' /etc/apache2/conf-enabled/other-vhosts-access-log.conf
```
Comment (JennyLawrance) [inline template-apache/Dockerfile]: do you need this? this means that this will also go to stderr: Please test this to make sure we don't have duplicate log entries.
